projects/iio_demo/builds.json

```diff
@@ -20,10 +20,13 @@
   },
   "stm32": {
     "iio": {
-      "flags": "IIO_EXAMPLE=y IIO_SW_TRIGGER_EXAMPLE=n IIO_TIMER_TRIGGER_EXAMPLE=n"
+      "flags": "IIO_EXAMPLE=y IIO_SW_TRIGGER_EXAMPLE=n IIO_TIMER_TRIGGER_EXAMPLE=n HARDWARE=sdp-ck1z.ioc"
+    },
+    "iio_usb_cdc_acm": {
```

Comment:
Also, we should use `RELEASE=y` for CI builds, since they won't be used for debugging. `-O2` might also reveal the use of undefined behavior.

Reply:
You are right.

I can add it, but I just figured, shouldn't the CI build all projects with RELEASE=y regardless of what's in the builds.json ? To me it would make sense.